monitoring/prometheus.md

```diff
@@ -40,6 +40,11 @@ Use our template to define a ServiceMonitor and to create a service account and
 oc process -f fuse-servicemonitor.yml -p NAMESPACE=<your-fuse-namespace> -p FUSE_SERVICE_NAME=<fuse-app-name> | oc apply -f -
 ```
 
+>**NOTE:** Make sure the configuration of the created service's selector is correct and matches all conditions properly.
+
+>**NOTE:** Fuse Console quickstart pods have no app name in labels. In the given example you have to remove *app: <app-name>* from the selector of the created service.
```

Comment:
What does Fuse console quickstart mean?  Can we add any link to the note?

Reply:
Is "Fuse Console quickstarts" creating apps from the openshift templates rather than from the maven archetypes?

Reply:
@tadayosi yes, we can add the link to Fuse quickstarts. Also, we can mention simply _quickstarts_.
@cunningt we use quickstarts from templates - https://github.com/jboss-fuse/application-templates/tree/master/quickstarts

Reply:
@jsolovjo I think we need to refer to these as "Fuse quickstart templates" or something similar to avoid confusion : https://access.redhat.com/documentation/en-us/red_hat_fuse/7.7/html-single/fuse_on_openshift_guide/index#create-project-s2i-source-workflow